hoomd/md/PotentialTersoff.h

```diff
@@ -82,18 +82,41 @@ class PotentialTersoff : public ForceCompute
         //! Destructor
         virtual ~PotentialTersoff();
 
-        //! Set the pair parameters for a single type pair
+        //! Set and get the pair parameters for a single type pair
```

Comment:
```suggestion
        //! Set the pair parameters for a single type pair
```
This method only sets the parameters.